sample_diagFl_key.json

```diff
@@ -0,0 +1,12 @@
+{
+  "type": "service_account",
+  "project_id": "sir2019-g7-sample-wfkciw",
+  "private_key_id": "74f6224b6ebc09264d4b81be31dca16879e64c7e",
+  "private_key": "-----BEGIN PRIVATE KEY-----\nMIIEvAIBADANBgkqhkiG9w0BAQEFAASCBKYwggSiAgEAAoIBAQDX5Ge5MVtc6xDf\nqqaxYtphz1ZZDI6FcSUsH71GHrIwMztAyYdb3PuiWGlEKtPJL+92c2SDsLz772Hq\nJWjTcHl5jmBcScsKESwzV7QrXieUCktFArW68VGtzhjzfKnRNnpUadoIgJry1tFl\nlNoSW0u9ERCrc8yVJba7u6m5O/v9k0dQmI3sCS2ITrY2SdzFelnNT2V8QRHwuh1Z\n27PbwWFFbOFDOUpvKb01SOnYIM2cWOFTexJwgeRpoBRz4gyyTXLHpus8eVlNwiUt\nY2YiLvUlXDbbp0O2JQcUDagU75HvmFOzCrS2HfDWIWQhUCretlLJJ1EhqdUv7b3s\nDiCS685xAgMBAAECggEAUGHBYMmIu9rlyHFQaUFmD/7TpoovUm3jKj+TCFWLgLjC\negfO9xwbMMH8R54uhnzris3CAeo/IwppCXXuRrNc9y5UoGf1SgVE0MT51x/SWqNO\njR5KLZWGFNFkptummrfO6jkTLhE8H+htxIWcgDMvyvBcpYmTBYEqPVHNCqPT2gBZ\nrJ2Gy0FeEMezSq/AucXYtdBhQix+wwD8jD7u8/oTnZYCMKIkTDNQzIn+ose9HBM8\nDek7OMkrgUzHo98w4au42UNqkKTabL8FKwLsv2Hzx5m+LyHxebLVNXp0+bNoW7zE\nNlvWAsY5xx4ItWgAXEHmgpRdm9/vSEtb7ZJjBZUhtQKBgQD53CqLO7BywaJP+GMe\n7LXlf36xHmZcolTRQIBJuYtH3il7LR2aXUBcemkJzC5cCMwKLAWzktliTGc0LKdu\nV7deyHrJC67DRRKqETECLCxZLXgKHQAh+kPXSyUYkbj9xxMJkaM8AFvXkUajgAyP\nZgvxB62uRS4SKowSiRTSfE0A8wKBgQDdMo1iysh9ptsFtOTeqzKGVOq+Tn9e+lzI\ncLwUe6Zkhm/lIVHobRCS2UqKBQ+O8ZSqaCqQ0KpyNEzaL7g2oIlQrdR63PCd1/2G\n8a9QVBGJae1TVVQgzc5gaLdvavP/91NHGN/LD5U1+EJLLKUgh91ksuS4OsuRalgB\nLOCEp4EsCwKBgBE7hhOkQj8n67HoR63SVZlGec5tclHhQEhOUorDKUADqoi8S9Zh\nqBa4upXaZzghwzDQaHRCJIwLCPMoQUPy+MqouQ80pxEAqpDgLqkfss9Tu3u3FFND\nl3o5zI0tP70BBecbDE8PF0Z+1ohgRH1/ZENzbB8g1lSaap68pZr+x7DRAoGAB3ge\niBh63qtUuAOcutwjD6AscRKrgGSJ34bXAwYN31Ue0Cjo4CVWqu7FcpWrKdOEhqCP\n12YEa2jSs4dq/AU9vciw5Rky0tGpUPY2SksccHxwNCEsfxMUBfNjRlLhPGJ4oGz5\nmZzEGBdy/aDbTpbn5e+Yg5QQra9em2ONjU0usnMCgYBJdYUSHoYTJSpbccGHCn8B\nwdB5m5SrIxVhON9VwetpXr1FvFBbQ23urlQAEFVOF8lyXAGY0LlYWTa1NEpWqopg\nzBj/bIEHIIG5dBJC3FB+7o7IHgONpVn80cVC2Asvq4c8oyY8szxm8ffHDb8XJHqu\n4cREIIs82kFmRuXQYkdIKQ==\n-----END PRIVATE KEY-----\n",
```

Comment:
HIDE THE KEYS! 🙈 _(even with a private rep)_

Reply:
How? in this case